sdk/eidolon_ai_sdk/agent/retriever_agent/retriever_agent.py

```diff
@@ -29,51 +30,58 @@ class RetrieverAgentSpec(RetrieverSpec):
         description="A detailed description of the the retriever including all necessary information for the calling agent to decide to call this agent, i.e. file type or location or etc..."
     )
     # these three fields are required and override the defaults of the subcomponents
-    loader_root_location: str = Field(None, description="A URL specifying the root location of the loader.")
+    loader_root_location: Optional[str] = Field(None, description="A URL specifying the root location of the loader.")
```

Comment:
Does it still make sense to leave this shorthand argument on the retriever? Since we are splitting things up now this seems like it will add more confusion than it helps via brevity.

Reply:
I was afraid of breaking other but I agree.